client/services.go

```diff
@@ -598,3 +599,12 @@ type WorkspacesClient interface {
 	DescribeWorkspaces(ctx context.Context, params *workspaces.DescribeWorkspacesInput, optFns ...func(*workspaces.Options)) (*workspaces.DescribeWorkspacesOutput, error)
 	DescribeWorkspaceDirectories(ctx context.Context, params *workspaces.DescribeWorkspaceDirectoriesInput, optFns ...func(*workspaces.Options)) (*workspaces.DescribeWorkspaceDirectoriesOutput, error)
 }
+
+//go:generate mockgen -package=mocks -destination=./mocks/backup.go . BackupClient
+type BackupClient interface {
```

Comment:
I know this isn't part of the original issue but I would add https://docs.aws.amazon.com/aws-backup/latest/devguide/API_ListProtectedResources.html @bbernays thoughts?

Reply:
I agree that we should add it. It definitely seems like information that would want to be searched

Reply:
For the sake of moving things forward, if you would prefer we can open up an issue to add support for this if we just want to go ahead and move this PR along

Reply:
Was an issue opened for this? @irmatov ?